src/iface/handle.rs

```diff
@@ -47,4 +48,26 @@ impl Nl80211InterfaceHandle {
     ) -> Nl80211InterfaceSetRequest {
         Nl80211InterfaceSetRequest::new(self.0.clone(), attributes)
     }
+
+    /// Add a new wireless interface.
+    ///
+    /// See <https://github.com/torvalds/linux/blob/v6.17/include/uapi/linux/nl80211.h#L374> for
+    /// required attributes.
+    pub fn add(
```

Comment:
`new` was already claimed (to create the handle), so `add` seemed the next best thing.